composer.json

```diff
@@ -12,7 +12,15 @@
         }
     ],
     "require": {
+        "php-64bit": "^7.0",
```

Comment:
While I do think that most to all systems should be 64bit by now, is this really a requirement at this point?